docs/contribute/source/plugin/wasi_nn.md

```diff
@@ -277,22 +277,29 @@ cmake --build build
 cmake --install build
 ```
 
-#### Apple Silicon Model
+### Appendix
```

Comment:
The main difference between Intel and Apple Silicon builds is the `-DWASMEDGE_PLUGIN_WASI_NN_GGML_LLAMA_METAL` flag, which is set to `OFF` for Intel and `ON` for Apple Silicon. Thus we cannot say `Apple Silicon Model` part is duplicate.

Reply:
The original has two identical `Apple Silicon Model` parts. One is in the MacOS section after `Intel Model` part and the other is in the Linux section after `General Linux without any acceleration framework` part.
I removed the one in the Linux section after `General Linux without any acceleration framework` part.
The diff is a little confusing.

Reply:
Screenshots from website:

The first one:

![image](https://github.com/WasmEdge/docs/assets/53310459/591a1fc7-19d5-4b65-8750-004f15d79aa0)

The second one:

![image](https://github.com/WasmEdge/docs/assets/53310459/efd44a69-c0d2-4ea4-8be0-1b348b094587)

Reply:
Maybe I should open another PR for this? because piper backend has not merged into wasmedge

Reply:
@PeterD1524 
Let's separate this into another PR. We would like to merge the Piper backend after merging the main PR.

Reply:
PR for fixing docs: https://github.com/WasmEdge/docs/pull/239